apps/i18n/javalab/en_us.json

```diff
@@ -34,5 +34,7 @@
   "saveToBackpack": "Save to Backpack",
   "twoMainMethods": "Error: Your code can only have one main method. We found at least two classes with main methods.",
   "backpackSaveInProgress": "Error: Save to Backpack is currently in progress, cannot start another Save to Backpack now.",
-  "unknownError": "Oops. We hit an error on our side. Try running your program again. If this error continues, contact us at support@code.org. Be sure to include the connection ID {connectionId} in your message.\n{type}"
+  "unknownError": "Oops. We hit an error on our side. Try running your program again. If this error continues, contact us at support@code.org. Be sure to include the connection ID {connectionId} in your message.\n{type}",
+  "backpackListLoadError": "An error occurred loading your backpack. Try again.",
+  "emptyBackpackMessage": "Files saved to your backpack will appear here."
```

Comment:
NIT: I may be wrong here, but I think we generally try to keep these files alphabetized by key. i.e. "emptyBackpackMessage" would go under "editor"

Reply:
yep, will fix